cmd/build/helmify/static/README.md

```diff
@@ -166,7 +166,8 @@ information._
 | emitAuditEvents                                            | Emit K8s events in configurable namespace for audit violations (alpha feature)                                                                                                                                                                                                                 | `false`                                                                                                                                                               |
 | enableK8sNativeValidation                                   | Enable the K8s Native Validating driver to create CEL-based rules  (alpha feature)                                                                                                                                                                                                                 | `false`                                                                                                                                                               |
-| vapEnforcement                                              | Generate K8s Validating Admission Policy resource. Allowed values are NONE: do not generate, GATEKEEPER_DEFAULT: do not generate unless label gatekeeper.sh/use-vap: yes is added to policy explicitly, VAP_DEFAULT: generate unless label gatekeeper.sh/use-vap: no is added to policy explicitly.  (alpha feature)                                                                                                                                                                                                                 | `GATEKEEPER_DEFAULT`                                                                                                                                                               |
+| vapEnforcement                                              | Generate K8s Validating Admission Policy resource. Allowed values are false: do not generate unless generateVAP: true is added to policy explicitly, true: generate unless generateVAP: false is added to policy explicitly. (alpha feature)                                                                                                                                                                                                                 | `false`                                                                                                                                                               |
```

Comment:
```suggestion
| vapEnforcement                                              | Generate K8s Validating Admission Policy resource. Allowed values are false: do not generate unless generateVAP: true is added to the constraint template explicitly, true: generate unless generateVAP: false is added to the constraint template explicitly. (beta feature)                                                                                                                                                                                                                 | `false`                                                                                                                                                               |
```

alpha -> beta : per last week's community call, we decided we can move this feature to beta.

Reply:
done